### ISSUE TITLE
Include Adaptive Sharpen algorithm with render scripts

### ISSUE THREAD

**Issue by aufkrawall** [zachsaw/MPDN_Extensions#174]
bacondither released his sharpen algorithm under BSD 2-clause license:
http://forum.doom9.org/showthread.php?t=172131
Since it's very usefull for post scaling sharpen (especially with NNEDI3), it'd be nice if it could be included with render scripts.

**Comment by Shiandow** [comment]
Found the problem. I'll fix it soon-ish.

**Comment by Shiandow** [comment]
Whoops wrong issue.

**Comment by trandoanhung1991** [comment]
I thought you can use it via Image Processor?

**Comment by aufkrawall** [comment]
Yes. But since it's open source, I think it makes sense to deliver it with the render scripts, just like SweetFX.
Probably Finesharp could also be bundled, but there aren't information about license.

**Comment by lanzorg** [comment]
Are you going to include it inside the mpdn installer?

**Comment by zachsaw** [comment]
I'll bring it into the repository.
